pkg/fixtures/purger_s3.go

```diff
@@ -0,0 +1,44 @@
+package fixtures
+
+import (
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/cloud/aws/s3"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	aws_s3 "github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type s3Purger struct {
+	logger     mon.Logger
+	client     s3.Client
+	bucketName *string
+}
+
+func newS3Purger(config cfg.Config, logger mon.Logger, settings *s3.Settings) *s3Purger {
```

Comment:
two other pointers are in
`s3FixtureWriter` struct. A pointer for settings and another one for the purger. I think we don't need them to be pointers. Do you approve?

Reply:
The when to user pointers is a tricky one.. We usually try to avoid pointers on basic types like int, string etc.. For structs we use pointers. You will find places where they are missing but usually we pass structs via reference and also have them as reference on fields.

Reply:
got it. that suits me, I am used to that as well.